src/views/v2/components/messages.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * View: Messages
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/messaging.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var array $messges An array of user-facing messages, managed by the View.
+ */
+
```

Comment:
:no_entry_sign: Missing @package tag in file comment

<sub>hash: e4b3bade4cedfcd8c3ffd3bfc4bb8edf</sub>